pkg/controller/v1beta1/inferenceservice/reconcilers/service/service_reconciler.go

```diff
@@ -134,13 +166,30 @@ func createService(componentMeta metav1.ObjectMeta, componentExt *v1beta1.Compon
 	return service
 }
 
+func createHeadSvc(componentMeta metav1.ObjectMeta) *corev1.Service {
+	predictorSvcName := componentMeta.Name
+	isvName := strings.TrimSuffix(componentMeta.Name, "-predictor")
+	componentMeta.Name = isvName + "-head"
+	service := &corev1.Service{
+		ObjectMeta: componentMeta,
+		Spec: corev1.ServiceSpec{
+			Selector: map[string]string{
+				"app": constants.GetRawServiceLabel(predictorSvcName),
+			},
+			ClusterIP:                "None",
```

Comment:
good catch. Yea I don't think it needs. I will remove it.

Reply:
Oops. I added it again.
because I understand why ClusterIP needs to be None. If it's not None, a port would be required, but since this service does not provide a specific function, there's no need to set a port. That's why I added None.